interfaces/python/CMakeLists.txt

```diff
@@ -15,9 +15,14 @@ get_directory_property(hasParent PARENT_DIRECTORY)
 if(NOT hasParent)
     message(STATUS "Standalone Python interface build")
 
+    # find_package(<PackageName>) searches prefixes given in <PackageName>_ROOT CMake variable
+    if(POLICY CMP0074)
+        cmake_policy(SET CMP0074 NEW)
+    endif()
+
     # Setup options
     option(HELICS_ENABLE_SWIG "Use SWIG to generate the Python interface files" ON)
-    option(HELICS_SWIG_GENERATE_INTERFACE_FILES_ONLY "Only generate the SWIG interface files" OFF)
+    #option(HELICS_SWIG_GENERATE_INTERFACE_FILES_ONLY "Only generate the SWIG interface files" OFF)
```

Comment:
Is this options still being defined somewhere else?  or is just a hidden option now?

Reply:
It's defined in https://github.com/GMLC-TDC/HELICS/blob/develop/interfaces/CMakeLists.txt#L58

I didn't want to remove the line entirely as a reminder that at some point refactoring the generate interface files only script to work standalone could be a simple change that could let it work as an example for someone making a 3rd party swig interface that doesn't live in the HELICS repository.